libtonode-tests/tests/sync.rs

```diff
@@ -77,8 +77,8 @@ async fn sync_test() {
     .await
     .unwrap();
 
```

Comment:
I thought we wanted to cut dbgs out?  I guess I am ok with commented out dbgs in test code.  Is that our practice?

Reply:
For sync development its good to have these dbgs ready to go when needed. I don't think we should normally allow debug but that's why I created a separate sync.rs test suite